### DISCUSSION TITLE
How can I setting up log request body to webhook.

### DISCUSSION THREAD

**Discussion by sisminnmaw** [healthchecks/healthchecks#820] [Q&A] [answered]
In my setting, I setup log data sent to healthcheck.io, and its showed in log request body.

attach log: https://healthchecks.io/docs/attaching_logs/

like this image:
![Capture](https://user-images.githubusercontent.com/5299365/230868594-dac185e6-4a4e-47e3-b1f2-45379879e6cc.PNG)

and then,
I wanna that request body sent to Webhook integrations. but I didnt found the ways to get that request body from Webhook integrations.
I found other $Name, $tag etc.

Thank you.

**Comment by cuu508** [comment, marked as answer]
Healthchecks supports a $BODY placeholder in webhook body template:

![image](https://user-images.githubusercontent.com/661859/231138332-f950ca0c-1277-4dbd-8a1a-f4b4737d6581.png)

**Comment by sisminnmaw** [comment]
@cuu508 
Thank you for reaching out to my question.

Currently I am using v1.20, there is no $BODY and $JSON in Supported Placeholders.
Now I found in change log its start from v2.4.

Thank you for your answer. Its help me a lot.
